bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html

```diff
@@ -47,8 +47,12 @@
 				<li ng-class="{active: page() == '/login'}" ng-show="user === undefined"><a href="#/login">Login</a></li>
 				<li ng-hide="user === undefined"><a href="" ng-click="logout()">Logout</a></li>
 			</ul>
-		<div class="pull-right"><h3 class="text-muted">Automated Benchmark Management</h3></div>
-		</div>
+		
```

Comment:
Please remove all the whitespace.

Reply:
I have created new 'images' folder and have placed the icons inside this folder. Also I have removed the whitespace. Please review the same.